src/formatters/deployResultFormatter.ts

```diff
@@ -285,6 +286,26 @@ export class DeployResultFormatter extends TestResultsFormatter implements Forma
   private displaySuccesses(): void {
     const successes = this.relativeFiles.filter(isSdrSuccess);
 
+    ensureArray(this.result.response.details.componentSuccesses)
+      .filter(
+        (fromServer) =>
+          // removes package.xml, other manifests
+          fromServer.componentType !== '' &&
+          // if we don't find the file in the response, it's because it doesn't exist locally, yet
+          !successes.find(
+            (fromLocal) => fromServer.fullName === fromLocal.fullName && fromServer.componentType === fromLocal.type
+          )
+      )
+      .map((s) =>
+        successes.push({
+          fullName: s.fullName,
+          // @ts-expect-error getState can return 'failed' which isn't applicable to FileSuccess
+          state: getState(s),
+          type: s.componentType ?? '',
+          filePath: 'Not found in project',
+        })
+      );
+
```

Comment:
IMO, SDR should do this.  Other clients may have the same problem and need this code.  I think SDR should report everything that happened, using the `ComponentSet` when possible to provide accurate file paths, but if not then use the file path from the response.  That means updating `MetadataApiDeploy.buildFileResponsesFromComponentSet()`.  @mshanemc - what are your thoughts?

Reply:
I tried [that](https://github.com/forcedotcom/source-deploy-retrieve/pull/1273), but NUTs had errors, maybe there's a different method, or a new one, but I don't think `getFileResponses` is it

Reply:
I like your SDR solution better (made 1 comment there).

I think the NUT failures look to all be `source-testkit` related, here: 
https://github.com/salesforcecli/source-testkit/blob/b5801a5ab89c044e94fd1ce14547ffe3dba9bf9f/src/assertions.ts#L120-L121

So maybe explore why that's hitting missing files?

Reply:
That would potentially break other consumers still